### ISSUE TITLE
Negative and High Plastic Strain Values

### ISSUE THREAD

**Issue by luchete80** [luchete80/WeldForm#57]
![image](https://user-images.githubusercontent.com/33092006/114034806-ce565e00-987e-11eb-9d70-1201be173756.png)

**Comment by luchete80** [comment]
```
	if (Fail == 1) {
		double J2	= 0.5*(ShearStress(0,0)*ShearStress(0,0) + 2.0*ShearStress(0,1)*ShearStress(1,0) +
						2.0*ShearStress(0,2)*ShearStress(2,0) + ShearStress(1,1)*ShearStress(1,1) +
						2.0*ShearStress(1,2)*ShearStress(2,1) + ShearStress(2,2)*ShearStress(2,2));
		//Scale back, Fraser Eqn 3-53
		ShearStress	= std::min((Sigmay/sqrt(3.0*J2)),1.0)*ShearStress;
		double dep=( sqrt(3.0*J2)- Sigmay)/ 3.*G;	//Fraser, Eq 3-49 TODO: MODIFY FOR TANGENT MODULUS = 0
		pl_strain+=dep;
	}

```

**Comment by luchete80** [comment]
Closed in b6c0c82cec08560c167d88dfed046e595c60752b
